woocommerce/woocommerce-admin
# Add onboarding profile REST endpoints

Fixes #1883

Adds endpoints for getting profile data and setting it.

**Profile params**
`account_type`
`store_address_1`
`store_address_2`
`store_zip`
`store_city`
`store_state`
`store_country`
`industry`
`product_types`
`product_count`
`selling_venues`
`other_platform`
`theme`
`purchase`

### Screenshots
<img width="412" alt="Screen Shot 2019-05-13 at 2 33 20 PM" src="https://user-images.githubusercontent.com/10561050/57600318-133ac280-758c-11e9-9a7a-f52df551ec27.png">

### Detailed test instructions:

1.  Make a GET request to `/wp-json/wc/v4/onboarding/profile`.
2.  All initial values should be present but set to null.
3.  Post to `/wp-json/wc/v4/onboarding/profile` with any of the profile params.
4.  Make another GET request to `/wp-json/wc/v4/onboarding/profile`.
5.  Note the updated fields.
